ASM/training-mode/Onscreen Display/Act OoJump/Aerial Out of Jump.asm

```diff
@@ -48,7 +48,7 @@ RedText:
     load r5, MSGCOLOR_RED
 
 DisplayText:
-    mr r3, 18                   # message kind
+    li r3, OSD.ActOoJump        # message kind
```

Comment:
`li` seems to be better than `mr`.
Because `mr` overwrite previous OSD even if OSD ID is different.

Reply:
I agree `li` is correct. However, this value is normally used to check against the current OSD settings. In this code the value is not used at all.

Reply:
Be aware that the numbers 0..32 are allowed as registers! So `mr r3, 18` is the same as `mr r3, r18`. `li` should be used for loading constant values.